showcase/app/templates/components/app-header.hbs

```diff
@@ -0,0 +1,175 @@
+
+        <:globalAfter>
+          <Shw::Placeholder @height="2em" @width="auto" @text="OrgPicker" />
+        </:globalAfter>
+
+        <:utility>
+          <Shw::Placeholder @height="2em" @width="auto" @text="Search" />
+          <Shw::Placeholder @height="2em" @width="auto" @text="HelpMenu" />
+          <Shw::Placeholder @height="2em" @width="auto" @text="GeoPicker" />
+          <Shw::Placeholder @height="2em" @width="auto" @text="UserMenu" />
+        </:utility>
+      </Hds::AppHeader>
+    </SG.Item>
+  </Shw::Grid>
+
+  <Shw::Divider @level="2" />
+
+  <Shw::Text::H3>With child component content</Shw::Text::H3>
+
+  <Shw::Grid @columns={{1}} {{style gap="2rem"}} as |SG|>
+    <SG.Item @label="With minimum recommended phase 1 content">
+      <Hds::AppHeader>
+        <:logo>
+          <Hds::AppHeader::HomeLink @icon="hashicorp" @ariaLabel="HashiCorp home menu" @href="#" />
+        </:logo>
+
+        <:globalAfter>
+          <Hds::Dropdown class="hds-side-nav__dropdown" as |dd|>
+            <dd.ToggleButton @text="Choose an organization" @icon="org" />
+            <dd.Checkmark>
+              organizationName
+            </dd.Checkmark>
+          </Hds::Dropdown>
+        </:globalAfter>
+
+        <:utility>
+          <Hds::Dropdown class="hds-side-nav__dropdown" @listPosition="bottom-right" as |dd|>
+            <dd.ToggleIcon @icon="help" @text="help menu" />
+            <dd.Title @text="Help & Support" />
+            <dd.Interactive @text="Documentation" @href="#" />
+            <dd.Interactive @text="Tutorials" @href="#" />
+            <dd.Interactive @text="Terraform Provider" @href="#" />
+            <dd.Interactive @text="Changelog" @href="#" />
+            <dd.Separator />
+            <dd.Interactive @text="Create support ticket" @href="#" />
+            <dd.Interactive @text="Give feedback" @href="#" />
+          </Hds::Dropdown>
+
+          <Hds::Dropdown class="hds-side-nav__dropdown" @listPosition="bottom-right" as |dd|>
+            <dd.ToggleIcon @icon="user" @text="user menu" />
+            <dd.Title @text="Signed In" />
+            <dd.Description @text="email@domain.com" />
+            <dd.Interactive @href="#" @text="Account Settings" />
+          </Hds::Dropdown>
+        </:utility>
+      </Hds::AppHeader>
+    </SG.Item>
+
+    <SG.Item @label="With max recommended phase 1 content">
+      <Hds::AppHeader>
+        <:logo>
+          <Hds::AppHeader::HomeLink @icon="hashicorp" @ariaLabel="HashiCorp home menu" @href="#" />
+        </:logo>
+
+        <:globalAfter>
+          <Hds::Dropdown class="hds-side-nav__dropdown" as |dd|>
+            <dd.ToggleButton @text="Choose an organization" @icon="org" />
+            <dd.Checkmark>
+              organizationName
+            </dd.Checkmark>
+          </Hds::Dropdown>
+        </:globalAfter>
+
+        <:utility>
+          <Hds::Button class="hds-side-nav__icon-button" @icon="search" @isIconOnly={{true}} @text="Search" />
```

Comment:
Icon only buttons **icon size** vs. Dropdown icon size needs to be more closely examined and perhaps reworked. (Examined & explored this and discussed with @jorytindall.)

I also am now realizing that the "one-off" `SideNav::Header::IconButton` was likely a mistake that we should also rethink. http://localhost:4201/components/side-nav#sidenav-header-iconbutton

We should look to come up with a better strategy for implementing the **dark "theme"** for the Button and Dropdown as well. (@jorytindall as a reminder)

Reply:
Not sure what you need as feedback here, so I'll go with my instinct:

The size of the icon in the IconButton appears to be the same as the icons on the dropdown": https://hds-showcase.vercel.app/components/side-nav#using-yielded-content
<img width="422" alt="screenshot_3857" src="https://github.com/hashicorp/design-system/assets/686239/36cc9620-49ea-4b19-afdb-865fe3b41354">

If instead of the `<Hds::Button>` I use the `<Hds::SideNav::Header::IconButton>` (that should be probably cloned into a `<Hds::AppHeader::IconButton>` component) this is what I see:
<img width="236" alt="screenshot_3858" src="https://github.com/hashicorp/design-system/assets/686239/77766ecd-4da8-4c18-8ef9-fb760a107217">
<img width="900" alt="screenshot_3859" src="https://github.com/hashicorp/design-system/assets/686239/38fb571c-7a38-44f0-98f3-9b1105b2e815">

Reply:
The `Hds::SideNav::Header::IconButton` forces a non-standard icon size in addition to adding dark theme styling. It was created to follow custom styling Terraform had been using for their search button. In my mind, needing a special component to change the button icon size highlights a possible design issue which should be solved in the base `Button` and `Dropdown` components. @jorytindall and I can look at this though and decide what to do.

This is the `AppHeader` using standard icon sizes:
<img width="1037" alt="image" src="https://github.com/hashicorp/design-system/assets/108769823/205238fa-11d9-440d-ac0f-48bea3af4342">